spatialprofilingtoolbox/db/importance_score_transcriber.py

```diff
@@ -0,0 +1,87 @@
+"""Ingest important scores and upload them to the local db."""
+
+from pandas import DataFrame, read_sql
+
+from spatialprofilingtoolbox.db.database_connection import DatabaseConnectionMaker
+from spatialprofilingtoolbox.db.create_data_analysis_study import (insert_new_data_analysis_study,
+                                                                   data_analysis_study_exists)
+from spatialprofilingtoolbox.workflow.common.export_features import ADIFeaturesUploader
+from spatialprofilingtoolbox.workflow.common.two_cohort_feature_association_testing import \
+    perform_tests
+from spatialprofilingtoolbox.standalone_utilities.log_formats import colorized_logger
+
+logger = colorized_logger(__name__)
+
+
+def transcribe_importance(df: DataFrame,
+                          target_var: str,
+                          database_connection_maker: DatabaseConnectionMaker,
+                          n_most_important: int = 1000) -> None:
+    """Upload importance score output from a cg-gnn instance to the local db.
+
+    Parameters:
+        df: DataFrame
+            One column, `importance_score`, indexed by `histological_structure`.
+        target_var: str
+            What variable the GNN was trained on to produce importance_score.
+        database_connection_maker: DatabaseConnectionMaker
+        n_most_important: int
+            Grab this many of the most important cells from each slide.
+    """
+    # Recover the study from the histological_structure
```

Comment:
I am also not a big believer in in-line comments, since they don't have a clear semantics as source code.
Here I would propose a refactor in which each of these 4 one-liner comments acting as "code block headers" are removed, and replaced with a function call to a function wrapping each block. E.g.:
```py
def recover_study_from_histological_structure(connection, histological_structure):
    return read_sql(f"""
        SELECT
            hsi.histological_structure,
            sdmp.study
        FROM histological_structure_identification hsi
            JOIN data_file df
                ON hsi.data_source=df.sha256_hash
            JOIN specimen_data_measurement_process sdmp
                ON df.source_generation_process=sdmp.identifier
        WHERE hsi.histological_structure={histological_structure}
        LIMIT 1
        ;
    """, connection)['study'][0]
```